tests/core/job/test_job_manager_with_sql_repo.py

```diff
@@ -51,6 +52,12 @@ def init_managers():
     _JobManagerFactory._build_manager()._delete_all()
 
 
+def clear_sql_session():
+    _build_engine.cache_clear()
+    _SQLSession._SessionLocal = None
+    _SQLSession._engine = None
```

Comment:
This method is used at standalone tests to clear the cache created by the previous test, so that the subprocess can connect properly to the database.